data/index.json

```diff
@@ -6,7 +6,7 @@
       "con-Latn-EC": "103 Singo a'i",
       "en": "Black People",
       "es": "El negro",
-      "_default": "Singo a'i"
+      "_default": "103"
```

Comment:
This change is incorrect - "Singo a'i" is a much better title for LingView to display than "103". On the old files, we used the edit.js script to modify some of the metadata that LingView displays, including this title, and it looks like those changes got lost as you removed and re-uploaded the files. It should be safe (and correct) to undo all the changes to this index.json file.

Reply:
Updated.